jsk_perception/sample/sample_imagenet_object_recognition.launch

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
It seems easier for user to find the sample if the sample files are named `sample_<node_name>.launch`, so my suggestion is that you add `imagenet_object_recognition.launch` in launch file with ros argument of `INPUT_IMAGE`.